audio/mpd/Portfile

```diff
@@ -28,76 +28,117 @@ master_sites        https://www.musicpd.org/download/${name}/${branch}/
 license             GPL-2+ LGPL-2.1
 use_xz              yes
 
-checksums           rmd160  c288933cd369d606e8fdfe49515c67cadc3da56e \
-                    sha256  503e5f9f237290f568ff7956ab2f9aed563594bf749f19b8fe994fb21434afea \
-                    size    806784
-
-depends_build \
-    port:pkgconfig
+checksums           rmd160  9951bf096e8d6d6fe3a07a82d660caf1d2bd91af \
+                    sha256  20a0ad01bf327b2dbeb6ae8e1af952cb0de83d2d63fab0fa4b7183a74765c201 \
+                    size    683768
 
 # this port only uses boost headers during build
 # and does not link with any boost libraries
 # no need to rebuild due to boost library ABI changes
 depends_build-append \
+    port:pkgconfig \
```

Comment:
This might be better left as a separate `depends_build-append` since the comment here is only explaining the `boost` dependency.